diff --git a/dist/transpilers/swc.js b/dist/transpilers/swc.js
index e49f88127820f7eda2b337e4f6e492f44c9828b3..e20af941a655c7eb789f8ba8cc48e0105109d9a6 100644
--- a/dist/transpilers/swc.js
+++ b/dist/transpilers/swc.js
@@ -174,14 +174,15 @@ function createSwcOptions(compilerOptions, nodeModuleEmitKind, swcInstance, swcD
                 parser: {
                     syntax: 'typescript',
                     tsx: isTsx,
-                    decorators: experimentalDecorators,
+                    decorators: true,
                     dynamicImport: true,
                     importAssertions: true,
                 },
                 target: swcTarget,
                 transform: {
                     decoratorMetadata: emitDecoratorMetadata,
-                    legacyDecorator: true,
+                    legacyDecorator: experimentalDecorators,
+                    decoratorVersion: experimentalDecorators ? '2021-12' : '2022-03',
                     react: {
                         throwIfNamespace: false,
                         development: jsxDevelopment,
diff --git a/dist-raw/README.md b/dist-raw/README.md
deleted file mode 100644
index 9eeaed31d53188a40567b6c9f6a33f52c389ad1f..0000000000000000000000000000000000000000
